notebooks/book2/05/minimize_kl_divergence.ipynb

```diff
@@ -0,0 +1,230 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #30.    latexify(width_scale_factor=1.2, fig_height=1.5, font_size=9)
</pre><p>Please don't touch font_size until extremely needed, you have only one figure and lot of space, you don't need to decrease font size from 12 to 9</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>